packages/apps/esm-implementer-tools-app/src/configuration/interactive-editor/config-subtree.component.tsx

```diff
@@ -12,12 +12,13 @@ export interface ConfigSubtreeProps {
 export function ConfigSubtree({ config, path = [] }: ConfigSubtreeProps) {
   function setActiveItemDescriptionOnMouseEnter(thisPath, key, value) {
     if (!implementerToolsStore.getState().configPathBeingEdited) {
+      const isLeaf = value.hasOwnProperty('_value') || value.hasOwnProperty('_type');
```

Comment:
Shouldn't this just be:

```suggestion
      const isLeaf = Object.hasOwn(value, '_value');
```

(`Object.hasOwn()` will also not fail in cases where `value` is something other than an object)